Content.Server/_NF/Speech/EntitySystems/ParrotSpeechSystem.cs

```diff
@@ -33,35 +39,58 @@ public override void Update(float frameTime)
             if (component.LearnedPhrases.Count == 0)
                 // This parrot has not learned any phrases, so can't say anything interesting.
                 continue;
+
+            if (component.RequiresMind && // imp 
+            !TryComp<MindContainerComponent>(uid, out var mind) | mind != null && !mind!.HasMind)
```

Comment:
indentation here - really confused me when i first looked at this. line breaking is a bit inconsistent as well - i get breaking once if the line is getting too long, but if the line break is this early then the if should probably have a separate line for each check